docs/index.md

```diff
@@ -88,6 +88,7 @@ examples/State Estimation Example.ipynb
 examples/Short Circuit Example.ipynb
 examples/Validation Examples.ipynb
 examples/Make Test Dataset.ipynb
+examples/Asymmetric Calculation Example.ipynb
```

Comment:
To the home page, please also add this (one of the subtasks in #447 ):

Add to the first lines of the PGM Docs (cfr. image below) that both symmetric and asymmetric calculations are supported
![294491447-d70338dc-e5d0-48e0-b135-df4f6a26f0f4](https://github.com/PowerGridModel/power-grid-model/assets/15234327/17b32de6-dd00-465d-9386-d11f86ec7f4e)